.github/workflows/everything.yml

```
@@ -105,6 +105,10 @@ jobs:
         GH_YML_MATRIX_OS: ${{ matrix.os }}
         GH_YML_MATRIX_COMPILER: ${{ matrix.compiler }}
         GH_YML_MATRIX_PARALLEL: ${{ matrix.parallel }}
+        CCACHE_BASEDIR: "${GITHUB_WORKSPACE}"
+        CCACHE_DIR: "${GITHUB_WORKSPACE}/.ccache"
+        CCACHE_COMPRESS: true
```

```suggestion
        CCACHE_COMPRESS: true
```

Ccache compress by default

---

What's the suggestion here?  Yours looks identical to mine.